### DISCUSSION TITLE
Having Words Instead of Numbers

### DISCUSSION THREAD

**Discussion by DOLLCAT-Hub** [openai/whisper#971] [Q&A]
Hello everyone! I have a corpus with some incorrect transcriptions. When I run Whisper on, it transcribes the numbers in their numeric form when I'd prefer it write them in their long alphabet form (example: 13=thirteen, 12th=twelfth, etc.). Any recommendations? Especially multilingual suggestions would be super helpful. Thank you!

**Comment by whicks1** [comment]
post process your outputs and run [num2words](https://pypi.org/project/num2words/) over it.

**Reply by DOLLCAT-Hub** [reply]
Would this work for multilingual examples as well, such as Russian or Korean?

Edit: My apologies, I didn't read the example. This is perfect!
